site/content/recipes/find_a_reviewer.md

```diff
@@ -0,0 +1,160 @@
+reviewing code is perceived as a waste of time by your colleagues, and you are
+hesitant to stick your neck out. If so, keep reading.
+
+<!--
+Probably need to add something about justifying time on code review to
+supervisor?
+--!>
+
+There are many other places to look for code reviewers. If you are fortunate
+enough to have a Research Software Engineering (RSE) group at your institution,
+then any member of that team would make a great reviewer for your code.
+These groups have many different ways of operating, so you will need to contact
+them to find out how to set up a code review. Forewarning: it is likely the
+time of the RSE will need to be covered under someone's budget, so if you were
+hesitant to have the conversation about code review in your research group,
+that problem will probably resurface here.
+
+Another great place to search within your institution or region are language or
+library-specific community groups (e.g. a Python or R User Group). This can
+help ensure the reviewer has experience with the same or similar tools you use.
+Before [making your advertisement](#contact), consider asking one of the
+community managers or someone you know in the community whether code review is
+something that has been done before or others might be interested in. Just as
+in the case with colleagues, you will be relying on people to volunteer their
+time, and as you are probably all to familiar with, researchers have that in
+very short supply. The community simply might not have the bandwidth to take on
+code review.
+
+It might be necessary to venture outside of your institution or immediate
+geographic region to find a reviewer, and it is an increasingly viable option
+in the context of remote working and many people getting used to asynchronous
+communication. If you are part of a research organisation, community, or
+working group that spans institutions and countries, this could be a viable
+avenue to search for a reviewer. Your connections in these areas might not
+themselves be suitable, but you can ask if they know anyone who might be.
+Open source or open science communities could similarly be leveraged,
+especially if you intend to make your software open source. If you have
+extended an open source library or even just made some convenient scripts to
+work with it, consider opening a merge/pull request to get the maintainers to
+have a look. A good example of this for more mature software is [rOpenSci's
+peer review initiative](https://bssw.io/blog_posts/a-community-of-practice-around-peer-review-for-long-term-research-software-sustainability).
+
+If none of the above options bear any fruit, then as a last resort you could
+make a post on the [Code Review Stack
+Exchange](https://codereview.stackexchange.com/). See the ["What questions can
+I ask about here?"](https://codereview.stackexchange.com/help/on-topic) page to
+determine whether it could suit your needs.
```

Comment:
I heard people can be a bit harsh on there. Anyone have experience with it?

Reply:
I personally don't have experience with this particular Stack Exchange site, but I agree that the perception of StackExchange is that many of the sub-sites can be quite intimidating and the communities are "harsh" on posters. It might be a good idea to add a warning here. I'll wait to hear from others in our group before doing so.